accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableinstaller/impl/AuthInstallerUserManagerPrefetchingImpl.java

```diff
@@ -140,6 +140,35 @@ private void refreshAuthorizableCacheIfNeeded(final String id, final Authorizabl
         }
     }
 
+    @Override
+    public void removeAuthorizable(final Authorizable authorizable) throws RepositoryException {
+        if (authorizable == null) {
+            return;
+        }
+
+        // remove auth from cache
+        authorizableCache.remove(authorizable.getID());
+        // if auth is a group, remove auth from the cache which lists groups of a given user
+        removeAuthorizableFromGroupsCache(authorizable);
+
+        authorizable.remove();
+    }
+
+    private void removeAuthorizableFromGroupsCache(final Authorizable authorizable) throws RepositoryException {
```

Comment:
I would rather name `removeGroup`, use `Group` as an argument type and do the instanceof check in the `removeAuthorizable`

Reply:
done!